app/assets/javascripts/discourse/app/components/header/home-logo.gjs

```diff
@@ -80,7 +80,7 @@ export default class HomeLogo extends Component {
   <template>
     <PluginOutlet
       @name="home-logo"
-      @outletArgs={{hash minimized=@minimized href=this.href}}
```

Comment:
We don't need `href` any more?

Reply:
Yes, I believe it does not make sense to pass down href in the plugin outlet since it's being set in the `home-logo` widget and not in the wrapper.